src/borg/archiver.py

```diff
@@ -3132,8 +3133,12 @@ def define_borg_mount(parser):
                               help='do not store ctime into archive')
         fs_group.add_argument('--nobirthtime', dest='nobirthtime', action='store_true',
                               help='do not store birthtime (creation date) into archive')
+        # --nobsdflags is misleading and deprecated.
+        # use --noflags as a replacement.
         fs_group.add_argument('--nobsdflags', dest='nobsdflags', action='store_true',
-                              help='do not read and store bsdflags (e.g. NODUMP, IMMUTABLE) into archive')
+                              help='DEPRECATED: do not read and store bsdflags (e.g. NODUMP, IMMUTABLE) into archive')
```

Comment:
instead of a source comment, rather point to --noflags from here, so it is visible on the UI.